app-ingest/src/main/scala/com/azavea/rf/JobDefinition.scala

```diff
@@ -0,0 +1,55 @@
+package com.azavea.rf
+
+import spray.json._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+
+import geotrellis.vector.Extent
+import geotrellis.vector.io._
+
+import java.net.URL
+
+/** Each TileSource, which sits the bottom of any JobDefinition (see below) is made of tiff urls and a target extent */
+case class TileSource(urls: Array[String], extent: Extent)
```

Comment:
@moradology I don't think we should rely on reading the tiff tags ourselves. If we externalize the band information into the job definition, then they can come from the tiff tags, or also alternate sources. This would also help us when we want to move away from only accepting GeoTiffs (e.g. Sentinel 2 data is JPG w/ metadata file)